[{"Owner":"cretz","Date":"2017-10-19T20:21:45Z","Content":"_lt_div class_eq__qt_mages_qt__gt_\n\t\t\t_lt_p_gt_\n\tReading _lt_a href_eq__qt_http_dd_//doc.babylonjs.com/overviews/using_the_physics_engine#angular-velocity_qt_ rel_eq__qt_external nofollow_qt__gt_http_dd_//doc.babylonjs.com/overviews/using_the_physics_engine#angular-velocity_lt_/a_gt_ it says to use a Quaternion_co_ but the code shows it accepts a Vector3 which is breaking my TypeScript code. It can_t_t type assert to a Vector3 because Quaternion doesn_t_t have toArray among other things. Does the func sig need to be updated or do the docs (and the example at _lt_a href_eq__qt_https_dd_//www.babylonjs-playground.com/#IGM3H_qt_ rel_eq__qt_external nofollow_qt__gt_https_dd_//www.babylonjs-playground.com/#IGM3H_lt_/a_gt_)?\n_lt_/p_gt_\n\n\t\t\t\n\t\t_lt_/div_gt_\n\n\t\t_lt_div class_eq__qt_ipsI_qt__gt__lt_/div_gt__lt_/div_gt_"}]